tools/python/util/optimize_onnx_model.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import pathlib
+
+from .onnx_model_utils import get_optimization_level, optimize_model
+
+
+def optimize_model_helper():
+    parser = argparse.ArgumentParser(f'{os.path.basename(__file__)}:{optimize_model_helper.__name__}',
+                                     description='''
+                                     Optimize an ONNX model using ONNX Runtime to the specified level.
+                                     See https://onnxruntime.ai/docs/performance/graph-optimizations.html for more
+                                     details of the optimization levels.'''
+                                     )
+
+    parser.add_argument('--opt_level', default='basic',
+                        choices=['disable', 'basic', 'extended', 'all'],
```

Comment:
>['disable', 'basic', 'extended', 'all'] [](http://example.com/codeflow?start=32&length=39)

would it be worth putting the optimization level list into a constant?

Reply:
just to avoid specifying it in multiple places (e.g., also in convert_onnx_models_to_ort.py)
though we also have the default values to consider, those could be constants too
it's fine as is, something to consider if we have more usages later